### DISCUSSION TITLE
Unable to index introduction.stories.mdx

### DISCUSSION THREAD

**Discussion by samoxxal** [storybookjs/storybook#25281] [Help]
### Summary

I have an old project that am trying to view and modify its component through the component explorer but everytime i try to run with 
`yarn storybook`

it opens up on browser with no components or modules showing up and with this error 

`ERROR in ./src/stories/Introduction.stories.mdx
Module build failed (from ./node_modules/@storybook/mdx2-csf/loader.js):
Could not parse expression with acorn: Unterminated template
 @ ./src/ lazy ^\.\/.*$ include: (?:\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)(?%21\.)(?=.)[^/]*?\.mdx)$ chunkName: [request] namespace object ./stories/Introduction.stories.mdx
 @ ./storybook-stories.js 10:11-14:5
 @ ./storybook-config-entry.js 6:0-50 25:21-29 28:2-31:4 28:58-31:3 30:31-39`

### Additional information

another error is shown up under that one is 

`preview compiled with 1 error
=> Failed to build the preview
99% end closing watch compilationWARN Force closed preview build
SB_BUILDER-WEBPACK5_0003 (WebpackCompilationError): There were problems when compiling your code with Webpack.
Run Storybook with --debug-webpack for more information.
    at starter (./node_modules/@storybook/builder-webpack5/dist/index.js:1:9660)
    at starter.next (<anonymous>)
    at Module.start (./node_modules/@storybook/builder-webpack5/dist/index.js:1:11692)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)`

**Comment by raul-wealth** [comment]
Try to rename `Introduction.stories.mdx`  to `Introduction.mdx`
